stage1/init/common/mount.go

```diff
@@ -18,12 +18,23 @@ import (
 	"fmt"
 	"os"
 	"strconv"
+	"syscall"
+
+	"github.com/coreos/rkt/pkg/fileutil"
+	"github.com/coreos/rkt/pkg/uid"
 
 	"github.com/appc/spec/schema"
 	"github.com/appc/spec/schema/types"
 	"github.com/hashicorp/errwrap"
 )
 
+// mountWrapper is a wrapper around a schema.Mount with an additional field indicating
+// whether it is an implicit empty volume converted from a Docker image.
+type mountWrapper struct {
+	mount          schema.Mount
```

Comment:
Just a suggestion, feel free to ignore - how about making `mountWrapper` a "subclass" of `schema.Mount`? I suppose it would have fewer changes (there would no need for changing `m.Foo` into `m.mount.Foo`).